src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+~ Licensed to the Apache Software Foundation (ASF) under one or more
+~ contributor license agreements. See the NOTICE file distributed with
+~ this work for additional information regarding copyright ownership.
+~ The ASF licenses this file to You under the Apache license, Version 2.0
+~ (the "License"); you may not use this file except in compliance with
+~ the License. You may obtain a copy of the License at
+~
+~      http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing, software
+~ distributed under the License is distributed on an "AS IS" BASIS,
+~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~ See the license for the specific language governing permissions and
+~ limitations under the license.
+-->
+<Configuration status="ERROR">
+  <Properties>
+    <Property name="LOG_DIR">${env:JENKINS_HOME}/logs</Property>
+  </Properties>
+  <Appenders>
+    <RollingRandomAccessFile name="audit" fileName="${LOG_DIR}/audit.log" filePattern="${LOG_DIR}/archive/audit.log.%d{yyyyMMdd_HHmmss}-%i">
+      <RFC5424Layout enterpriseNumber="38143" includeMDC="true" mdcId="RequestContext" appName="Log4j-audit-plugin" mdcPrefix="" newLine="true"/>
+      <Policies>
```

Comment:
You can use a bare `SizeBasedTriggeringPolicy` here instead of wrapping it in a `Policies` element (which allows you to configure multiple policies into a composite).

Reply:
was contemplating on that - was thinking that we might want to use other policies later on.

Reply:
Fine by me. We might end up using the cron-based policy, especially if we want to make that configurable in the UI as well (there are examples of cron-like config UIs in Jenkins elsewhere).